### PR TITLE
REV: Add high-level paper summary

### LINKED ISSUE #161: JOSS paper comments
_Issue related to JOSS review: https://github.com/openjournals/joss-reviews/issues/4248_

The current version of the [JOSS paper](https://github.com/jsheunis/connectomemapper3/blob/master/paper/paper.md) is well written and clearly explains the need and functionality of CMP3. I have a few comments:
- While reading through the paper content, I adapted the wording slightly in a few places. These are (in my opinion) minor improvements, and you can use your discretion to decide whether or not to include these changes. I'll submit a small PR shortly.
- The JOSS guidelines state that the paper should have _"A summary describing the high-level functionality and purpose of the software for a diverse, non-specialist audience"_. At the moment, the summary section makes out the bulk of the paper, and I consider much of it to be quite detailed and specialist. I don't think this content is a problem though, just that I'm missing a shorter summary for a *diverse, non-specialist audience*. Perhaps a new paragraph could be included to serve as such a summary, while the current **Summary** section could be renamed to **Functionality** (or something similar). That is if you (and the JOSS editors) agree that this is indeed a useful approach.
- The **Mention** section's title seems a bit out of context, perhaps rename it to **Software use** or **Ongoing research** or **Community impact**, or something along those lines.

### PR DESCRIPTION
**Pull request by sebastientourbier**
PR related to JOSS review: https://github.com/openjournals/joss-reviews/issues/4248

This PR resolves #161.

It addresses  issue point 2) by adding in the paper the suggested high-level summary. 

It addresses issue points 2 and 3 by renaming the following sections:
- "Summary" -> "Design and functionality overview"
- "Mention" -> "Community impact"

Finally, the sections "Outputs ready to be reused in the BIDS ecosystem" and "A focus on accessibility and versatility" have been swapped to improve the rendering of the compiled PDF.

Generation of the PDF of the most updated draft could done with https://whedon.theoj.org/ as follows:
Screen Shot 2022-06-02 at 10 40 24

### CODE REVIEW

**Comment by sebastientourbier** [comment]
Dear co-authors/contributors,

Hope you are doing well!

CMP3's review is on going, and while I can take care of most of the points that are very software/docs related, I would need your feedback on this.

In fact in response to the following comment:
> The JOSS guidelines state that the paper should have "A summary describing the high-level functionality and purpose of the software for a diverse, non-specialist audience". At the moment, the summary section makes out the bulk of the paper, and I consider much of it to be quite detailed and specialist. I don't think this content is a problem though, just that I'm missing a shorter summary for a diverse, non-specialist audience. Perhaps a new paragraph could be included to serve as such a summary, while the current Summary section could be renamed to Functionality (or something similar). That is if you (and the JOSS editors) agree that this is indeed a useful approach.
I added a short high-level summary in the paper that I asked you to review whenever you find a little bit of time, and add comments / suggestions.

Best

File: paper/paper.md [hunk 1]
```diff
@@ -75,6 +75,27 @@ bibliography: paper.bib
 # aas-journal: Astrophysical Journal <- The name of the AAS journal.
 ---
 
+# Summary
+
+Connectome Mapper 3 (CMP3) is an open-source processing pipeline software, written in Python 3, for multi-scale multi-modal connectome mapping
+  of the human brain.
+It provides researchers with an unique workflow, implemented in accordance to the BIDS App framework
```
[the remark this answers is not in the document]
Comment (sebastientourbier) [reply paper/paper.md:82]:
```suggestion
It provides researchers with a unique workflow, implemented in accordance to the BIDS App framework
```

File: paper/paper.md [hunk 2]
```diff
@@ -75,6 +75,27 @@ bibliography: paper.bib
 # aas-journal: Astrophysical Journal <- The name of the AAS journal.
 ---
 
+# Summary
+
+Connectome Mapper 3 (CMP3) is an open-source processing pipeline software, written in Python 3, for multi-scale multi-modal connectome mapping
```
Comment (jwirsich) [inline paper/paper.md:80]: Connectome Mapper 3 (CMP3) is an open-source processing pipeline software (add:) *for neuroimaging data* (...),

Comment (jwirsich) [reply paper/paper.md:80]: Maybe add this or something similar: The connectome refers to a representation of the brain as a graph derived from both structural or functional connectivity between a pair of brain regions.

**Review by jwirsich — APPROVED** [review]

File: paper/paper.md [hunk 3]
```diff
@@ -75,6 +75,27 @@ bibliography: paper.bib
...
+It provides researchers with a unique workflow, implemented in accordance with the BIDS App framework
+  [@GorgolewskiBIDSMethods:2017], that leverages a number of widely adopted software tools to map a connectome
+  representation of the structural and functional brain systems at multiple scales.
+The workflow can take any structural / diffusion / resting-state functional MRI dataset structured accordingly
+  to the BIDS standard [@GorgolewskiTheExperiments:2016], and is intended to be run via its Docker and
+  Singularity/Apptainer containerized versions.
+To improve its accessibility, CMP3 comes with a graphical user interface, which supports
+  the user in all the steps involved in the configuration of the pipelines, the configuration and execution of
+  the BIDS App, and the control of the output quality.
+CMP3 has been successfully employed in a number of research papers, and is now being currently extended to
```
Comment (mschoettner) [inline paper/paper.md:91]:
```suggestion
CMP3 has been successfully employed in a number of research papers, and is currently being extended to
```

File: paper/paper.md [hunk 4]
```diff
@@ -75,6 +75,27 @@ bibliography: paper.bib
...
+  electroencephalography, to offer a solid multi-modal framework to the community for the investigation of
+  brain network function and organization at specific scales, as well as a map that links different spatial
+  and temporal scales.
+CMP3 is available from the [Python Package Index (PyPI)](https://pypi.org/project/connectomemapper/),
+  and the container images available from [DockerHub](https://hub.docker.com/r/sebastientourbier/connectomemapper-bidsapp)
```
Comment (mschoettner) [inline paper/paper.md:96]:
```suggestion
  and the container images are available from [DockerHub](https://hub.docker.com/r/sebastientourbier/connectomemapper-bidsapp)
```

**Comment by sebastientourbier** [comment]
Dear @jsheunis,

Please find in this PR a revised version of the paper which now includes a more high-level summary as well as the renaming of a few sections to integrate your suggestions / comments in #161.

I will merge it when you approve. Please let me know what you think.

Here is a recap of the PR description which more fully details the changes:

> PR related to JOSS review: [openjournals/joss-reviews#4248](https://github.com/openjournals/joss-reviews/issues/4248)
> 
> This PR resolves #161.
> 
> It addresses issue point 2) by adding in the paper the suggested high-level summary.
> 
> It addresses issue points 2 and 3 by renaming the following sections:
> 
>     * "Summary" -> "Design and functionality overview"
> 
>     * "Mention" -> "Community impact"
> 
> 
> Finally, the sections "Outputs ready to be reused in the BIDS ecosystem" and "A focus on accessibility and versatility" have been swapped to improve the rendering of the compiled PDF.
> 
> Generation of the PDF of the most updated draft could done with https://whedon.theoj.org/ as follows: Screen Shot 2022-06-02 at 10 40 24

Thank you very much in advance.
Best regards

**Review by jsheunis — APPROVED** [review]
I like these changes, the summary contains everything that is important to mention about the package, communicated at a high level for a broader audience.

**Comment by sebastientourbier** [comment]
Great! I will then take care of the merge :)
I also like these changes that you suggested.
Thank you very much @jsheunis!

**Pull request merged**